cppfd/mesh.cpp

```diff
@@ -133,3 +133,38 @@ void Mesh::write_vtk(std::string file_name){
   std::cout<<std::endl;
   std::cout<<"Visualization file created: \""<<file_name<<"\""<<std::endl;
 }
+
+void Mesh::write_txt(std::string file_name, std::string output_type){
+  uint64_t nx = this->n_cells_x;
+  uint64_t ny = this->n_cells_y;
+
+  ofstream file;
+  file.open(file_name);
+
+  if(!file){
+    std::cout<<"Error in creating text file"<<std::endl;
+  }
+
+  if(output_type == "velocity"){
+    for(int j = ny; j > - 1; j--){
```

Comment:
for consistency you should use `uint64_t` (or `auto`) in the loops

Reply:
@pierrepebay this is because if the variable is now a `uint64_t`, it will  **never** be greater than -1 because in that case, -1 is cast to the positive value of  2^64 - 1... (the `u` in `uint64_t` stands for **u**nsigned)